### ISSUE TITLE
Char escaping doesn't follow Sass 3.3 or 3.4 ways

### ISSUE THREAD

**Issue by kaelig** [sass/libsass#1271]
See http://sassmeister.com/gist/cd6d8d263d555bca6cf9

Neither of these things return the correct value in LibSass:
```scss
$character-code: f102;

test {

  /* Expected: "\f102" */

  /* Sass 3.4 */
  content: unquote("\"\\#{$character-code}\"");

  /* Sass 3.3 */
  content: str-slice("\x", 1, 1) + $character-code;

}
```

**Comment by kaelig** [comment]
If a workaround exists I'd be interested to know about it.

**Comment by HugoGiraudel** [comment]
On top:

```scss
$value: str-slice(unquote('\\'), 1);
```

`/` in Ruby Sass, `//` in LibSass.

**Comment by HugoGiraudel** [comment]
@kaelig: This should do the trick on LibSass:

```scss
.foo {
  content: #{'\"\\' + $character-code + "\""};
}
```

**Comment by kaelig** [comment]
Thanks Hugo, although LibSass renders the actual unicode char (`content: ;`). Is that what is supposed to happen? Does it work everywhere?

**Comment by HugoGiraudel** [comment]
Reported in #1231.

**Comment by JLHwung** [comment]
Another situation might be relevant to this issue, see http://sassmeister.com/gist/6b58d3d5b01bb2a932c6

This snippet is from bootstrap-sass-official and it breaks the apperance of breadcrumb component.

**Comment by xzyfer** [comment]
Fixed via #1534
